### PR TITLE
Setup a base CI tests

### PR DESCRIPTION
**Pull request by micpiotrowski**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+jobs:
+  super-lint:
+    name: Check markdown files with remark
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo npm install --global remark-cli remark-lint-no-dead-urls
```
Comment (PawelLipski) [inline .github/workflows/test.yml:12]: Hmm how about `remark-validate-links` (for internal links)? Coz `remark-lint-no-dead-urls` IIRC checks external links only (?)

Comment (micpiotrowski) [reply .github/workflows/test.yml:12]: But we doesn't have any internal links (and rather will not have)

File: ci_checks/emoji/conf.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from models import *
 
 MD_FILES_PATH = 'src/assets/tips'
+EMOJI_FILES_PATH = 'src/assets/emoji'
 
 FORBIDDEN_EMOJI = [
```
Comment (PawelLipski) [inline ci_checks/emoji/conf.py:6]: Wow nice <3 haven't seen that before

Comment (PawelLipski) [reply ci_checks/emoji/conf.py:6]: `PoliticallyForbiddenEmoji('krakow', emoji_instead='machete'),` whoops it's the reverse ;)

File: ci_checks/emoji/check.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+from glob import glob
+from os.path import isfile
+from os import listdir
+import sys
+import re
+
+from conf import MD_FILES_PATH, EMOJI_FILES_PATH, FORBIDDEN_EMOJI
+
+
+map_of_forbidden_emoji = {emoji.emoji_name: emoji for emoji in FORBIDDEN_EMOJI}
```
Comment (PawelLipski) [inline ci_checks/emoji/check.py:11]: Nit: I'd call it `forbidden_emoji_by_name`

Comment (PawelLipski) [reply ci_checks/emoji/check.py:11]: To make it clear what's the key and what's the value

File: ci_checks/emoji/check.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+emoji_regex = re.compile(r'!\[(a?)\]\((.*?)\)')
+
+is_error = False
+
+emoji_filename_list = []
```
Comment (PawelLipski) [inline ci_checks/emoji/check.py:17]: Maybe `used_emoji_filenames` to make it clear what's in the list

File: ci_checks/emoji/conf.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+from models import *
+
+MD_FILES_PATH = 'src/assets/tips'
+EMOJI_FILES_PATH = 'src/assets/emoji'
+
+FORBIDDEN_EMOJI = [
+    PoliticallyForbiddenEmoji('biedux', emoji_instead='linux'),
+    PoliticallyForbiddenEmoji('krakow', emoji_instead='machete'),
+    PoliticallyForbiddenEmoji('fashtag'),
```
Comment (PawelLipski) [inline ci_checks/emoji/conf.py:9]: Pls include `nie` as well ;)

**Review by PawelLipski — APPROVED** [review]
Nice 🎉

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**